### ISSUE TITLE
syntax/css.vim needs update

### ISSUE THREAD

**Issue by 71GA** [vim/vim#5459]
I took the latest css.vim syntax file and put it into **/usr/share/vim/vim81/syntax/** but some keywords remain uncolored. For example: 

1.  If i have multiple attributes split over multiple lines:

```
background: url("./images/index/004.jpg")
no-repeat
center
fixed;
```
here for example `fixed` is colored diferently _(wrong)_ than `no-repeat` and `center` _(right)_.

2. or for example with this keywords: 

```
backdrop-filter: blur(5px);
```
where in the first line `backdrop-filter` should be colored consistently, while only `filter` is colored while `backdrop` is not. 

3. or this: 

```
transition: backdrop-filter 2s;
```
where `backdrop-` is colored diferently than `filter`. 

**QUESTION:** Is there anything that can be added to the `css.vim` in order to add some keywords manually or could anyone of the developer team do it?

**Comment by chrisbra** [comment]
did you contact the maintainer?

**Comment by 71GA** [comment]
I did today.

**Comment by albfan** [comment]
FTR: https://github.com/JulesWang/css.vim

**Comment by chrisbra** [comment]
closing as fixed as of e7b1ea0276cc83cd5c612f3189a174a60d57b5ed

Note, upstream css syntax script is: https://github.com/jsit/css.vim
